superbench/benchmarks/micro_benchmarks/ort_inference_performance.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""TensorRT inference micro-benchmark."""
+
+import time
+import statistics
+from pathlib import Path
+
+import torch
+import torchvision.models
+import numpy as np
+import onnxruntime as ort
+from onnxruntime.quantization import quantize_dynamic
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform, Precision
+from superbench.benchmarks.micro_benchmarks import MicroBenchmark
+
+
+class ORTInferenceBenchmark(MicroBenchmark):
+    """ONNXRuntime inference micro-benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._pytorch_models = [
+            'resnet50',
+            'resnet101',
+            'resnet152',
+            'densenet169',
+            'densenet201',
+            'vgg11',
+            'vgg13',
+            'vgg16',
+            'vgg19',
+        ]
```

Comment:
Do we have bert/gpt/lstm ? Actually I have the same question on Yifan's PR as well.

Reply:
I ever thought you guys have reached the agreement about what models to support in Yifan's PR.....

Reply:
For other models like GPT, the model file size will be big. We can discuss what models to support in the future.

Reply:
Got it.